AzureCalling/External/Calling/AudioDeviceType.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+import UIKit
+
+enum AudioDeviceType: String {
+    case receiver = "iPhone"
```

Comment:
Ok; in this case, we'd probably need the View Controller to decide what to return instead of the enum (The enum can return just `receiver`).  I'd suggest this be a future ticket, since detection of the iOS device type has not been done yet.